packages/website/next.config.js

```diff
@@ -66,6 +66,11 @@ module.exports = withTypescript(
               Props: ['pretty-proptypes', 'default'],
             }),
           );
+          config.plugins.push(
```

Comment:
Does the `ProvidePlugin` take multiple arguments/values on its object?

If so, can we extend the existing plugin instead of adding a second one?